docs/uperf.md

```diff
@@ -189,6 +198,55 @@ To enable Multus in Ripsaw, here is the relevant config.
       ...
 
 ```
+### Scale
+Scale in this context refers to the ability to enumerate UPERF 
+client-server pairs during test in a control fashion using the following knobs.
+
+`colocate: true` will place each client and server pod pair on the same node.
+
+`density_range` to specify the range of client-server pairs that the test will iterate.
+
+`node_range` to specify the range of nodes that the test will iterate.
+
+`step_size` to specify the incrementing method.
+
+Here is one scale example:
+
+```
+      ...
+      pin: false
+      colocate: false
+      density_range: [1,10]
+      node_range: [1,128]
+      step_size: log2
+      ...
+```
+Note, the `scale` mode is mutual exlusive to `pin` mode with the `pin` mode has higher precedence.
```

Comment:
"mutually"

Reply:
also with the `pin` mode "having" higher precendence.

Reply:
good checks